requirements.txt

```diff
@@ -14,3 +14,4 @@ pandas
 quandl
 yfinance
 retworkx>=0.5.0
+dataclasses; python_version < '3.7'
```

Comment:
This has been added to requirements but not to setup.py? It should be added there too I think.

Reply:
Sorry. I forget about setup.py. I added it. Thank you.